edgelet/iotedge/src/check/mod.rs

```diff
@@ -1964,22 +1964,26 @@ fn is_rfc_1035_valid(name: &str) -> bool {
             Some(c) => c,
             None => return false,
         };
-        if first_char < 'a' || first_char > 'z' {
+        let first_char_invalid =
+            (first_char < 'a' || first_char > 'z') && (first_char < 'A' || first_char > 'Z');
```

Comment:
Could this be replaced by  `is_ascii_alphabetic()`?

Reply:
Oh yeah, good point. Done.